### DISCUSSION TITLE
Management proto issue

### DISCUSSION THREAD

**Discussion by andrewmcone** [bnhf/openvpn-admin-plus#72] [Q&A]
I have installed OVAP via Portainer and everything seems to be working but I constantly see MAPPING ERROR
I clearly tried to use 172.20.0.1, 172.17.0.1, 127.0.0.1 and even my external server IP but still MAPPING ERROR.

What I should to in order to make it working?
Please kindly help.

**Comment by bnhf** [comment]
@andrewmcone 

> What I should to in order to make it working?

Have you run through _Configuration - OpenVPN Config_ once, inputting your values of course, so the Management Interface (MI) gets setup? Your OpenVPN server will need to be restarted after that initialization process.

In Configuration - Settings, using the IP of your Docker gateway (usually 172.17.0.1) is the preferred IP to use in the interest of not exposing the MI to your LAN, but you can also use the LAN IP of your server if such exposure is not a concern.

**Reply by andrewmcone** [reply]
Thank you very much for replying!
I have added management 127.17.0.1 2080 to config file. I also restarted several times portainer and docker will all containers. 
I am using exactly same gateway as you mentioned.

However, I think where is the issue. 
I have openpvn server installed /etc/opevnpn/server/ (there is also client folder /etc/openvpn/client) but OVAP looking by some reason into /etc/openvpn/ instead of real one. 
I have of course defined where OVAP should look into config but I see OVAP created by some reason server.conf file in /etc/openvpn/

Previously there was this error in logs
cp: cannot create regular file '/etc/openvpn/easy-rsa/vars': No such file or directory
OpenVPN directory set to: /etc/openvpn
Working directory set to: /opt


I have updated/defined everything for vars and then error disappered. 
P.S. I also see /etc/openvpn/ set as default folder in Portainer Volums of OVAP. For this purpose I created symlink to /etc/openvpn/server/
Maybe this will help somehow to help me.
